.github/.wordlist.txt

```diff
@@ -695,6 +694,7 @@ ipynb
 IPython
 ISCAN
 isHexString
+IsInitiator
```

Comment:
IsInitiator is not a word

Reply:
It believe was previously introduced by other PR, the diff is the result of running `sort -f | unique -i`. I can remove it